fleetd/fleet.go

```diff
@@ -48,7 +48,9 @@ func main() {
 		os.Exit(1)
 	}
 
-	if *printVersion {
+	var args = userset.Args()
```

Comment:
Yes, you are right, I have modified as per your suggestion. Please let me know whether there is any other issue or not.